### ISSUE TITLE
Issue with too long of a restriction

### ISSUE THREAD

**Issue by Nayruden** [TeamUlysses/ulx#107]
See https://forums.ulyssesmod.net/index.php/topic,10503.msg53433 for discussion. Could be an issue with passing it through the console.

**Comment by SticklyMan** [comment]
Yes, I can confirm this is caused by the 255 or so character limit on console commands. D:
